### PR TITLE
[FIX][10.0] Fixes #138 - accounts for false values in sale_crm_medical_prescription

### LINKED ISSUE #138: sale_crm_medical_prescription: error when creating lead
I got this error while creating a lead in sale_crm_medical_prescription:

File "/media/sf_vertical-medical/sale_crm_medical_prescription/wizards/medical_lead_wizard.py", line 78, in action_create_leads
    'name': ', '.join(l.name for l in order)
TypeError: sequence item 0: expected string, bool found

### PR DESCRIPTION
**Pull request by kellylougheed**

### CODE REVIEW

File: sale_crm_medical_prescription/wizards/medical_lead_wizard.py [hunk 1]
```diff
@@ -75,7 +75,9 @@ def action_create_leads(self, ):
                 'pharmacy_id': self.pharmacy_id.id,
                 'prescription_order_line_ids': order_lines,
                 'is_prescription': True,
-                'name': ', '.join(l.name for l in order)
+                'name': ', '.join(
+                    [l.name if l.name is not False else '' for l in order]
```
Comment (lasley) [inline sale_crm_medical_prescription/wizards/medical_lead_wizard.py]: `l.name or '' for l in order`

**Comment by kellylougheed** [comment]
Done @lasley

File: sale_crm_medical_prescription/wizards/medical_lead_wizard.py [hunk 2]
```diff
@@ -75,7 +75,7 @@ def action_create_leads(self, ):
                 'pharmacy_id': self.pharmacy_id.id,
                 'prescription_order_line_ids': order_lines,
                 'is_prescription': True,
-                'name': ', '.join(l.name for l in order)
+                'name': ', '.join([l.name or '' for l in order])
             })
 
         _logger.debug('Created %s', lead_ids)
```
Comment (lasley) [inline sale_crm_medical_prescription/wizards/medical_lead_wizard.py]: Let's remove this line, it's lowering our test coverage & doesn't really seem needed in long term

**Review by lasley — APPROVED** [review]
👍  outside of the line removal in my most recent comment

**Comment by kellylougheed** [comment]
Change made @lasley

**Comment by lasley** [comment]
@kellylougheed - looks like tests are failing on last commit

**Comment by kellylougheed** [comment]
Fixed @lasley! Should I rebase on release/10.0 to try to fix the conflicts?

**Comment by lasley** [comment]
> Should I rebase on release/10.0 to try to fix the conflicts?

Yeah your best bet here is to squash all the commits here into one first, which will make the rebase much easier. 

Typically we wouldn't be able to do this, but we're all on the same team & our interest is getting shit done vs worrying about our own attributions, so I am 👍  for the ease in this case.

**Comment by kellylougheed** [comment]
Conflicts resolved!

**Review by lasley — APPROVED** [review]

**Pull request merged**